docs/pages/machine-id/reference/telemetry.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Telemetry
+description: An explanation of the telemetry collected by Machine ID
+---
+
+This document explains what telemetry is collected by the Machine ID `tbot`
+agent, why we want to collect this telemetry and, how to opt in or out.
+
+## Why?
+
+Machine ID is an emerging part of the Teleport product and it's helpful for us
+to be able to identify the kinds of use-cases people have. This allows us to
+prioritise more common usages. Whilst we try to collect this sort of information
+by talking to users directly, having a more general overview of the product in
+the wild helps us make even more informed decisions and avoid our decisions
+being solely influenced by a select few users.
+
+## Anonymous telemetry
+
+Anonymous telemetry is currently the only mode supported by `tbot`. This means
+that the collected data does not include anything which identifies:
+
+- The specific machine `tbot` is running on,
+- The cluster `tbot` is connecting to,
+- The hosts, applications, databases and kubernetes clusters `tbot` connects to,
+- The user that has configured `tbot`.
```

Comment:
I would make the punctuation consistent for the items in this list, since it's not a sentence.

Reply:
[86b9402](https://github.com/gravitational/teleport/pull/22541/commits/86b940214b51ee65df6a400fdce8aaf76edcd77a)